src/encoded/audit/genetic_modification.py

```diff
@@ -0,0 +1,34 @@
+from snovault import (
+    AuditFailure,
+    audit_checker,
+)
+
+
+@audit_checker('GeneticModification', frame='object')
+def audit_genetic_modification_reagents(value, system):
+    '''
+    Genetic modifications with missing reagents will be flagged by this audit.
+    Modifications with acceptable substitutes for reagents will not be flagged,
+    depending on their method. See the method dependecy in file.json for more
+    detail.
+    '''
+    missing_reagents = False
+
+    if not value.get('reagents'):
+        method = value['method']
+        if method == 'CRISPR':
+            if not value.get('guide_rna_sequences'):
+                missing_reagents = True
+        elif method == 'RNAi':
+            if not value.get('rnai_sequences'):
+                missing_reagents = True
+        elif method == 'TALEN':
+            missing_reagents = True
+        elif not value.get('documents'):
+            missing_reagents = True
+
```

Comment:
No need for spaces like this in python.  The required indents create "natural" visual cues, so we don't need spaces in general.